modules/tensor_mechanics/test/tests/rom_stress_update/verification.i

```diff
@@ -0,0 +1,305 @@
+    direction = right
+  [../]
+  [./temp_fcn]
+    type = PiecewiseConstant
+    data_file = ss316_verification_data.csv
+    x_index_in_file = 0
+    y_index_in_file = 5
+    format = columns
+    xy_in_file_only = false
+    direction = right
+  [../]
+
+  [./rhom_soln_fcn]
+    type = PiecewiseConstant
+    data_file = ss316_verification_data.csv
+    x_index_in_file = 0
+    y_index_in_file = 7
+    format = columns
+    xy_in_file_only = false
+    direction = right
+  [../]
+  [./rhoi_soln_fcn]
+    type = PiecewiseConstant
+    data_file = ss316_verification_data.csv
+    x_index_in_file = 0
+    y_index_in_file = 8
+    format = columns
+    xy_in_file_only = false
+    direction = right
+  [../]
+  [./creep_rate_soln_fcn]
+    type = PiecewiseConstant
+    data_file = ss316_verification_data.csv
+    x_index_in_file = 0
+    y_index_in_file = 10
+    format = columns
+    xy_in_file_only = false
+    direction = right
+  [../]
+
+  [./rhom_diff_fcn]
+    type = ParsedFunction
+    vars = 'rhom_soln rhom'
+    vals = 'rhom_soln rhom'
+    value = 'abs(rhom_soln - rhom) / rhom_soln'
+  [../]
+  [./rhoi_diff_fcn]
+    type = ParsedFunction
+    vars = 'rhoi_soln rhoi'
+    vals = 'rhoi_soln rhoi'
+    value = 'abs(rhoi_soln - rhoi) / rhoi_soln'
+  [../]
+  [./creep_rate_diff_fcn]
+    type = ParsedFunction
+    vars = 'creep_rate_soln creep_rate'
+    vals = 'creep_rate_soln creep_rate'
+    value = 'abs(creep_rate_soln - creep_rate) / creep_rate_soln'
+  [../]
+[]
+
+
+[Modules/TensorMechanics/Master]
+  [./all]
+    strain = FINITE
+    add_variables = true
+    generate_output = 'vonmises_stress'
+    use_automatic_differentiation = true
+  [../]
+[]
+
+[BCs]
+  [./symmx]
+    type = ADPresetBC
+    variable = disp_x
+    boundary = left
+    value = 0
+  [../]
+  [./symmy]
+    type = ADPresetBC
+    variable = disp_y
+    boundary = bottom
+    value = 0
+  [../]
+  [./symmz]
+    type = ADPresetBC
+    variable = disp_z
+    boundary = back
+    value = 0
+  [../]
+  [./pressure_x]
+    type = ADPressure
+    variable = disp_x
+    component = 0
+    boundary = right
+    function = vmJ2_fcn
+    constant = 0.5e6
+  [../]
+  [./pressure_y]
+    type = ADPressure
+    variable = disp_y
+    component = 1
+    boundary = top
+    function = vmJ2_fcn
+    constant = -0.5e6
+  [../]
+  [./pressure_z]
+    type = ADPressure
+    variable = disp_z
+    component = 2
+    boundary = front
+    function = vmJ2_fcn
+    constant = -0.5e6
+  [../]
+[]
+
+[Materials]
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 1e11
+    poissons_ratio = 0.3
+  [../]
+  [./stress]
+    type = ADComputeMultipleInelasticStress
+    inelastic_models = rom_stress_prediction
+  [../]
+  [./rom_stress_prediction]
+    type = ADLAROMCreepStressUpdate
+    temperature = temperature
+    effective_inelastic_strain_name = effective_creep_strain
+    internal_solve_full_iteration_history = true
+    apply_strain = false
```

Comment:
This option seems odd to me, and I"m not following all the functions as well as I'd like. Could I ask you to walk me through the procedure you are using in your verification tests here please?

Reply:
I am forcing the direct application of the inputs to check the outputs (mobile and immobile dislocations, and old creep strain), isolated from the finite element solve. This checks the numbers directly from the ROM.